apps/src/templates/teacherNavigation/EmptySection.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import {NavLink} from 'react-router-dom';
+
+import {LinkButton} from '@cdo/apps/componentLibrary/button';
+import {Heading3, BodyTwoText} from '@cdo/apps/componentLibrary/typography';
+import emptyDesk from '@cdo/apps/templates/teacherDashboard/images/empty_desk.svg';
+import blankScreen from '@cdo/apps/templates/teacherDashboard/images/no_curriculum_assigned.svg';
+import i18n from '@cdo/locale';
+
+import {TEACHER_NAVIGATION_PATHS} from './TeacherNavigationPaths';
+
+import styles from './teacher-navigation.module.scss';
+import dashboardStyles from '@cdo/apps/templates/teacherDashboard/teacher-dashboard.module.scss';
+
+interface EmptySectionProps {
+  hasStudents: boolean;
+  hasCurriculumAssigned: boolean;
+  element: React.ReactElement;
+}
+
+const EmptySection: React.FC<EmptySectionProps> = ({
```

Comment:
I cannot help but feel this is misnamed - like it feels more like `TeacherNavContainer` - because it is either going to show graphics related to an `EmptySection` or all the appropriate content for a non-empty-section...

Reply:
I think you are right, can we also do that in my next PR linked above?